feature-libs/product/configurators/common/components/configure-product/configure-product.component.ts

```diff
@@ -12,27 +10,9 @@ import { CommonConfigurator } from '../../core/model/common-configurator.model';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ConfigureProductComponent {
-  productContext: OutletContextData;
+  product$: Observable<Product> = this.currentProductService.getProduct();
```

Comment:
`getProduct()` loads the default full-blown scope `ProductScope.DETAILS`. But in `ConfigureProductComponent ` we need only a small scope of the product data `ProductScope.CONFIGURATOR`. What about calling `getProduct(ProductScope.CONFIGURATOR)` for performance? 

Note: The performance benefit will occur only when user is on a page that doesn't need full data (i.e. but only `ProductScopes.LIST` scope - which is smaller than `DETAILS`). And asking here in the `ConfigureProductComponent`  for full `DETAILS` would cause unnecessary bigger payload in the backend response.

Needed change: I think it's justified then to add field `code` to the occ-endpoint product scope configuration for `CONFIGURATOR`

Reply:
Yes, makes sense...Thanks for that hint